docs/architecture/decisions/0008-allow-non-signon-users-to-access-fact-check-view-via-a-jwt.md

```diff
@@ -0,0 +1,69 @@
+
+# 8. Allow non-signon users to access fact check view via a JWT
+
+Date: 2025-11-27
+
+Status: Accepted
+
+## Context
+
+In the [previous ADR], we proposed adding a fact check view as a Rails engine to allow users who do not have access to 
+signon to preview content securely.
+
+We needed to determine how to authenticate these external users without requiring signon credentials.
+
+## Decision
+
+We will use JSON Web Tokens (JWTs) for authentication, following the existing pattern used for draft content access in 
+the draft stack.
+
+### How it works
+
+1. **Token generation**: Each Edition will have a method to generate a JWT containing:
+    - The content ID
+    - A new UUID field on the edition (identifying the specific edition)
+    - A timestamp indicating when the token expires
+
+2. **Token validation**: When a user visits the fact check view URL with a JWT query parameter:
+    - The JWT is decoded and validated
```

Comment:
By "valid" do we mean that it is technically correct, e.g. as conforming to https://datatracker.ietf.org/doc/html/rfc7519#section-7.2

Reply:
or does it also include verifying that the token has not:
- expired
- been revoked

?